source/Octopus.Tentacle.Contracts/ScriptServiceV3Alpha/KubernetesAgentScriptExecutionContext.cs

```diff
@@ -5,7 +5,7 @@ namespace Octopus.Tentacle.Contracts.ScriptServiceV3Alpha
     public class KubernetesAgentScriptExecutionContext : IScriptExecutionContext
     {
         [JsonConstructor]
-        public KubernetesAgentScriptExecutionContext(string image, string? feedUrl, string? feedUsername, string? feedPassword)
+        public KubernetesAgentScriptExecutionContext(string? image, string? feedUrl, string? feedUsername, string? feedPassword)
```

Comment:
technically the JSON value for `image` may not exist, so this should be nullable